### PR TITLE
Migrate to Coveralls & Added support versions

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Added support versions
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| How to test?      | N/A
| Possible impacts? | N/A

### CODE REVIEW

**Review by atomiix — APPROVED** [review]

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -32,7 +34,14 @@ jobs:
...
-
-    - name: Code Coverage
-      run: php vendor/bin/codacycoverage clover build/clover.xml
+      run: ./vendor/bin/phpunit --coverage-clover build/clover.xml
+
+    - name: Upload coverage results to Coveralls
+      if: matrix.php-versions == '7.1'
+      env:
+        COVERALLS_RUN_LOCALLY: 1
+        COVERALLS_REPO_TOKEN: 4PY3vFXVlLE7HkKNbEDA5JMaeOeIwGhqX
```
Comment (PierreRambaud) [inline .github/workflows/php.yml]: Should it really be there :thinking:

Comment (matthieu-rolland) [reply .github/workflows/php.yml]: Is it a use-case for encrypted secrets ?

https://docs.github.com/en/actions/reference/encrypted-secrets

Comment (Progi1984) [reply .github/workflows/php.yml]: Fixed.

**Comment by PierreRambaud** [comment]
Thank you! Tests were failing because Coveralls token is not shared between forks and the main repository

**Pull request merged**